contents/blog/how-to-run-transparent-company.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2022-07-26
+title: "How to run a transparent startup"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+There really hasn't been a single situation where transparency hasn't been to our company's advantage.
+
+Most companies seem to share when necessary but not by default. I hope this piece changes that - as a startup, you've done something wrong if you don't trust your own team.
```

Comment:
```suggestion
Most companies seem to share when necessary but not by default. I hope this piece changes that because, as a startup, you've done something wrong if you don't trust your own team.
```